_layouts/doc.html

```diff
@@ -2,6 +2,13 @@
 {% include menu.html %}
 <main class="doc">
   <article class="container">
+    <div class="alert alert-warning hidden alert-outdated" role="alert">
+      <strong class="header">Heads up!</strong>
+      <p>
+        This document is for an old version of ESLint that is <em>no longer supported</em>.
+        Please <a class="alert-link" href="{{ site.url }}/docs/user-guide/migrating-to-4.0.0">upgrade</a> to a newer release!
```

Comment:
I don't mind doing this, but I think people typically just use the current version of the docs anyway, and only use the versioned URL when they deliberately need to look at the docs for an older version. So the message might not be useful to many people since they would already know they're using an unsupported version.